mighty/mighty_models/sac.py

```diff
@@ -121,17 +138,26 @@ def forward(
           mean: Gaussian mean
           log_std: Gaussian log std
         """
-        feats = self.feature_extractor(state)
-        x = self.policy_net(feats)
+        x = self.policy_net(state)
```

Comment:
Not in the mighty format. The separate feature extractor is there to have a predictable structure and access to a feature embedding "Mighty-er" format would be to have a feature extractor -> policy head and then a q_feature_extractor. No functional difference, but it's relevant for continuity between algos.

Reply:
Updated -- performance similar